backend-tests/tests/test_tenantadm.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import pytest
+import subprocess
+import time
+
+import api
+
+from common import mongo, clean_mongo
+from infra.cli import CliTenantadm
+from common import User, Device, Tenant, create_user, create_tenant, \
+        create_tenant_user, create_organization
+
+logger = logging.getLogger(__name__)
+
+class TestCreateOrganizationCLIEnterprise:
+    api_mgmt_useradm = api.client.ApiClient(api.useradm.URL_MGMT)
+    logger = logger.getChild("TestCreateOrganizationCLIEnterprise")
+
+    def test_success(self, clean_mongo):
+        """
+        Create a single organization and verify that the created user
+        is able to log in.
+        """
+        self.logger.info("Starting `test_success`")
+
+        out = create_organization("fooCorp", "user@example.com", "pwd")
+        self.logger.debug("tenant id: %s" % out)
+
+        # Retry login for 5 min
+        for i in range(60 * 5):
```

Comment:
you are retrying for 5 minutes here and for 3 minutes in other places;

Reply:
I think 5 minutes is a little overkill, made all login attempts retry for 3 minutes.